ydb/core/tx/datashard/reshuffle_kmeans.cpp

```diff
@@ -385,6 +385,9 @@ void TDataShard::HandleSafe(TEvDataShard::TEvReshuffleKMeansRequest::TPtr& ev, c
         rowVersion = GetMvccTxVersion(EMvccTxMode::ReadOnly);
     }
 
+    LOG_N("Starting TReshuffleKMeansScan " << record.ShortDebugString()
```

Comment:
I think it's already written somewhere here with debug

Reply:
Didn't found in TTxDataShardPrefixKMeansScan.BuildToPosting logs, so let's better just keep it

https://paste.yandex-team.ru/0bd38707-e420-486b-8f2c-f21867355be8/text

An example of existing log has much less info:
```
2025-03-24T20:10:29.840712Z node 2 :BUILD_INDEX DEBUG: Create TPrefixKMeansScan Id: 1 Parent: 40 Child: 41 UploadTable:  K: 2 Clusters: 0 State: SAMPLE Round: 0 / 300 PostingBuf size: 0 PrefixBuf size: 0 UploadBuf size: 0
```